app/views/modules/_header.html.erb

```diff
@@ -10,31 +10,51 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
+<div class="container-fluid grey">
+
 <div id="masthead" class="container">
   <div class="row visible-xs">
     <div class="lib-logo col-xs-8">
       <%= ualib_logo_link %>
     </div>
+    <div class="col-xs-4 user-util">
+        <%= render partial: "modules/user_util_dropdowns"%>
+      </div>
   </div>
   <div class="row">
     <div class="lib-logo col-sm-3 hidden-xs">
       <%= ualib_logo_link %>
     </div>
-    <div class="mast-branding col-sm-5">
-      <%= era_av_logo_link %>
-    </div>
-    <div class="col-sm-4">
-      <form accept-charset="UTF-8" action="/catalog" id="search-form-header" method="get" role="search" class="form-inline hidden-xs mast-sm-search">
-        <div style="display:none"><input name="utf8" type="hidden" value="✓"></div>
-        <input id="search_field" name="search_field" type="hidden" value="all_fields">
-        <div class="form-group">
-          <input type="text" class="form-control q" id="search-field-header" name="q" placeholder="enter search terms">
-        </div>
-        <button type="submit" class="btn btn-default sm-search"><span class="glyphicon glyphicon-search" aria-hidden="true"></span>Search ERA Audio+Video</button>
-      </form>
-      <div class="col-md-12 user-util hidden-xs">
+    
+      <div class="mast-branding col-sm-5">
+        <h2 class="hidden-xs">education &amp; research archive</h2>
+        <h1><a href="/">ERA</a></h1>
+        <h3><span class="glyphicon glyphicon-triangle-right" aria-hidden="true"></span>AUDIO <span class="glyphicon glyphicon-plus logo-plus" aria-hidden="true"></span> VIDEO</h3>
+      </div>
+      <div class="col-sm-4 user-util hidden-xs">
         <%= render partial: "modules/user_util_dropdowns"%>
       </div>
+     
+          <div class="col-xs-12 mobile-search visible-xs">
+ <form accept-charset="UTF-8" action="/catalog" id="search-form-header" method="get" role="search" class="form-inline">
```

Comment:
Once again - this was copy and pasted from existing code (I believe it was written by someone at Indiana University). 

https://github.com/ualbertalib/avalon/blob/master/app/views/modules/_header.html.erb

I don't mean to be skiving off on this work, but I also don't really feel that it's my place to rewrite code. My sense was I was brought in to make the UI look good by hooking into existing code...